10up/classifai
# Fix: Pages are not being scanned by Language Processing

### Description of the Change

- Add page to rest route check (`is_rest_route`).
- Add a filter (`classifai_rest_bases`) to support custom post type with custom rest base.
- Remove unused enqueue.
- Remove deprecated (and removed) function (`is_gutenberg_page`).

### Benefits

Pages now are scanned and tagged by Language Processing.

### Verification Process

1. Edit a page then hit the Update button.
2. See the page updated normally without any error.
3. Go back to the pages list, see the page processed by Waston.

### Checklist:






### Applicable Issues
#161
